build_tools/cirrus/arm_tests.yml

```diff
@@ -17,4 +17,9 @@ linux_aarch64_test_task:
     folder: /root/.conda/pkgs
     fingerprint_script: cat build_tools/cirrus/py39_conda_forge_linux-aarch64_conda.lock
 
-  test_script: bash build_tools/cirrus/build_test_arm.sh
+  test_script: |
+    bash build_tools/cirrus/build_test_arm.sh
+    bash build_tools/cirrus/update_tracking_issue.sh true
```

Comment:
I'm still not super familiar with our CI infrastructure -- what is the intent of the addition of
```
    bash build_tools/cirrus/update_tracking_issue.sh true
```

Reply:
If the script passes, `update_tracking_issue.sh true` runs which updates a tracking issue stating that test is passing. When the script fails, then `on_failure` is triggered and runs `update_tracking_issue.sh false`, which creates or updates a tracking issue stating that the job failed.

An example of an issue that `update_tracking_issue.sh` opened is: https://github.com/scikit-learn/scikit-learn/issues/26745

Reply:
Would this help?
```suggestion
    # On success, this script is run updating the issue.
    bash build_tools/cirrus/update_tracking_issue.sh true
```